src/testing.py

```diff
@@ -266,9 +266,9 @@ def execute(tester, inputs, fails, exclusions=None, attributes=None):
     sys.stdout.write(
         '\n'.join(['Ran %s tests.' % len(results),
                    'Got %s successes.' % len(successes),
-                   'Got %s failures.' % len(failures)]))
+                   'Got %s failures.' % len(failures)]) + '\n')
     if failures:
-      sys.stdout.write('\nUnexpected failures:\n')
+      sys.stdout.write('Unexpected failures:\n')
```

Comment:
Why delete the newline in the beginning?

Reply:
I added it onto previous print statement instead.. Now each statement is repsonsible for ending its own line :)